test/mock-connector.py

```diff
@@ -38,18 +30,14 @@
 
 EVENT_DATA = 'data.publish'
 
-channel.exchange_declare(exchange=fog_exchange, durable=True,
-exchange_type='topic')
-channel.exchange_declare(exchange=cloud_exchange, durable=True,
-exchange_type='topic')
+KEY_UPDATE = 'data.update'
+KEY_REQUEST = 'data.request'
 
-result = channel.queue_declare('', exclusive=True)
-queue_name = result.method.queue
+logging.basicConfig(
+    format='%(asctime)s %(message)s',
+    level=logging.INFO,
+    datefmt='%Y-%m-%d %H:%M:%S')
 
-channel.queue_bind(
-        exchange=cloud_exchange, queue=queue_name, routing_key='device.*')
-channel.queue_bind(
-        exchange=cloud_exchange, queue=queue_name, routing_key='data.*')
 
 def callback(ch, method, properties, body):
```

Comment:
Since we have more than one callback. Rename it

Reply:
done